src/OrleansServiceBus/Providers/Streams/EventHub/EventHubCheckpointer.cs

```diff
@@ -22,16 +19,16 @@ internal class EventHubPartitionCheckpoint
         private Task inProgressSave;
         private DateTime? throttleSavesUntilUtc;
 
-        public bool Exists { get { return entity != null && entity.Offset != EventHubConsumerGroup.StartOfStream; } }
+        public bool Exists => entity != null && entity.Offset != EventHubConsumerGroup.StartOfStream;
```

Comment:
We aren't using the hosted build I'm VSO. We have a big pool of agents and servers to deploy to,  to run all the load tests and functionals. It's just a matter of installing vs2015 in the build agents,  not big deal,  but we haven't done it yet